src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -78546,26 +78546,35 @@ class C
 {
     void M()
     {
-        var lam1 = (object o = null) => {}; // 1
+        var lam1 = (object o = null) => {};
         var lam2 = (object o = null!) => {};
-        var lam3 = (object o = default) => {}; // 2
+        var lam3 = (object o = default) => {};
         var lam4 = (object o = default!) => {};
         var lam5 = ([AllowNull] string s = null) => {};
-        var lam6 = ([DisallowNull] string? s = null) => {}; // 3
+        var lam6 = ([DisallowNull] string? s = null) => {};
         var lam7 = ([DisallowNull] string? s = "a") => {};
     }
 }
 """;
             CreateCompilation(new[] { source, AllowNullAttributeDefinition, DisallowNullAttributeDefinition }, options: WithNullableEnable()).VerifyDiagnostics(
                 // (6,32): warning CS8625: Cannot convert null literal to non-nullable reference type.
-                //         var lam1 = (object o = null) => {}; // 1
+                //         var lam1 = (object o = null) => {};
                 Diagnostic(ErrorCode.WRN_NullAsNonNullable, "null").WithLocation(6, 32),
                 // (8,32): warning CS8625: Cannot convert null literal to non-nullable reference type.
-                //         var lam3 = (object o = default) => {}; // 2
+                //         var lam3 = (object o = default) => {};
                 Diagnostic(ErrorCode.WRN_NullAsNonNullable, "default").WithLocation(8, 32),
+                // (10,20): warning CS8622: Nullability of reference types in type of parameter 's' of 'lambda expression' doesn't match the target delegate '<anonymous delegate>' (possibly because of nullability attributes).
```

Comment:
>warning CS8622: Nullability of reference types in type of parameter 's' of 'lambda expression' [](http://example.com/codeflow?start=28&length=94)

Does this mean we do not copy nullability attributes to the inferred delegate type? (Just confirming the expected behavior.)

Reply:
Yes.